src/components/LoginForm.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Button, Grid, Link, Stack, Typography, TextField } from '@mui/material';
+const LoginInput = () => {
+  return (
+    <Grid spacing={2} container item direction="column" xs>
+      <Grid item container direction="column" spacing={5} sx={{ textAlign: 'center', alignItems: 'center' }}>
+        <Grid item xs>
+          <Typography variant="h2" fontWeight={'bold'} sx={{ marginLeft: 'auto' }}>
+            Login
+          </Typography>
+        </Grid>
+        <Grid item xs sx={{ width: '70%' }}>
+          <Stack spacing={2}>
+            <TextField type="text" label="Email" />
+            <TextField type="password" label="Password" />
+          </Stack>
+        </Grid>
+        <Grid item xs>
+          <Stack spacing={2}>
+            <Button variant="contained" sx={{ width: '15em' }}>
```

Comment:
These buttons should be the lime-ish color like in the figma. I saw your messages on discord about the theming though, so not sure if it's necessary.

It'd be cool if you hardcoded it in for now though, would look nicer than it currently does and it would be a good reference to what the final page would look like in the end.

Reply:
@TheRizz told me I should use the default mui color scheme for now until theming gets properly set up.